testdata/README.md

```diff
@@ -0,0 +1,21 @@
+# Test data
+
+The JSON files within this directory hiearchy represent test data
+for the schema in the repository. The directory structure matches
+that in the [proto](../proto) directory, namely the protobuf package
+hierarchy.
+
+Each JSON file should be named according to the data message it
```

Comment:
At first I didn't realize the naming aligns to the protobuf message / "Data messages" field as seen in the README.

Can we say something like the "according to the corresponding protobuf's `message`"

Reply:
Changed to "CloudEvent data protobuf message" which I think its unambiguous.